src/Loader/GherkinFileLoader.php

```diff
@@ -72,7 +73,7 @@ public function load(mixed $resource)
     {
         $path = $this->getAbsolutePath($resource);
         if ($this->cache) {
-            if ($this->cache->isFresh($path, filemtime($path))) {
+            if (($lastModified = filemtime($path)) !== false && $this->cache->isFresh($path, $lastModified)) {
```

Comment:
(PHPStan) filemtime might fail and return false, but we need an int at this point.